### ISSUE TITLE
quicklook doesnt follow links

### ISSUE THREAD

**Issue by namoran** [QL-Win/QuickLook#978]
I have a folder of shortcuts to files that are in 2 child folders. I tried to view the files with quicklook on the shortcuts but that didn't work. It worked fine on the files themselves. 

my purpose for doing this on the shortcuts versus the files themselves is that the files must be kept in separate folders becuase they come from different sources online, however, they are dated in the first part of the name and I want to see them in chronological order, so I put links to all the files in the parent folder and sorted them by name. Files are named like this 20080101_testingresults.pdf

**Comment by rabelux** [comment]
Ok so what you are saying is that you have a feature request to change the behaviour of previewing links. Instead of a link you want to see a preview of the target, right?

**Comment by namoran** [comment]
Yes that’s right, currently it does not preview the link (I imaging that would be some text file) or the file itself. Nothing happens

**Comment by xupefei** [comment]
AFAIK previewing the true file behind a shortcut is already supported. No idea why it didn't work for you.

**Comment by xupefei** [comment]
Could you try more shortcuts and if it happens with all of them, fill in the issue template?

**Comment by rabelux** [comment]
Just tested it and previewing shortcuts does indeed show the target file.

**Comment by namoran** [comment]
Could it because I’m working on a remote pc?

Nicholas A. Moran, PG
CoFounder & Geologist
www.envirotechsummit.org

**Comment by emako** [comment]
Any issue about PDF, use https://github.com/QL-Win/QuickLook.Plugin.PdfViewer-Native as an alternative, MS will fix it.
